frontend/src/components/MapView/LeftPanel/ChartsPanel/DateSlider/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { Box, Typography, createStyles, makeStyles } from '@material-ui/core';
+import React from 'react';
+import RangeSlider from 'react-range-slider-input';
+import 'react-range-slider-input/dist/style.css';
+import './slider.css';
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    sliderContainer: {
+      width: 'calc(100% - 4rem)',
```

Comment:
Is there a reason you're using a calc instead of setting it to 100% and having 2rem margins on each side?